modules/ols-disabling-data-collection-operator.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/about/ols-about-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-disabling-data-collection-operator_{context}"]
+= Disabling data collection on the {ols-long} Service
+
+The {ols-long} Service collects information about the questions you ask and the feedback you provide on the answers that the Service generates. You can disable data collection by modifying the `OLSConfig` custom resource.
+
+.Prerequisites
+
+* You have an LLM provider available for use with the {ols-long} Service.
+
+* You have installed the {ols-long} Operator.
+
+* You have deployed the {ols-long} Service.
+
+.Procedure
+
+. Open the {ols-long} `OLSConfig` custom resource (CR) file by running the following command:
++
+[source,terminal]
+----
+$ oc edit olsconfig cluster
+----
+
+. Modify the `spec.ols.userDataCollection` field to disable data collection for the {ols-long} CR.
++
+[source,yaml]
+----
+apiVersion: ols.openshift.io/v1alpha1
+kind: OLSConfig
+metadata:
+  name: cluster
+spec:
+  ols:
+    userDataCollection:
+      feedbackDisabled: true # <1>
+      transcriptsDisabled: true # <2>
+----
+<1> Disables feedback collection 
+<2> Disables transcript collection.
+
+. Click *Save*. 
```

Comment:
`oc edit olsconfig cluster` opens the config in VIM. There is no `save` button.
One needs to do ESC->type ":wq"->ENTER.
Maybe it would be better to just say "save the changes" without specifing how.

Reply:
done